quantum/pointing_device/pointing_device_auto_mouse.c

```diff
@@ -267,6 +275,11 @@ void pointing_device_task_auto_mouse(report_mouse_t mouse_report) {
         if (!layer_state_is((AUTO_MOUSE_TARGET_LAYER))) {
             layer_on((AUTO_MOUSE_TARGET_LAYER));
         }
+#ifdef AUTO_MOUSE_ONESHOT
```

Comment:
```suggestion
#ifdef POINTING_DEVICE_AUTO_MOUSE_ONESHOT
```

Reply:
This feels a bit weird to me, as this is really an option, a bit like AUTO_MOUSE_DELAY / AUTO_MOUSE_THRESHOLD / ...

Reply:
yeah, some of the naming isn't exactly consistent with the feature, in general. :/